Popup/Popover: accessibility (#5778)
DevExpress/devextreme-documentation: concepts/05 UI Components/Popover/03 Accessibility/15 Screen Reader Support.md

    })
    export class AppComponent {
        onShowingHandler(e) {
            const overlayContent = e.component.content().parentElement;
            overlayContent.setAttribute("role", "dialog");
        }
    }

##### Vue

    <!-- tab: App.vue -->
    <template>
    <DxPopover ...
        @showing="onShowingHandler"
    />
    </template>

    <script>
    import { DxPopover } from 'devextreme-vue/Popover';

    export default {
        components: {
            DxPopover
        },
        methods: {
            onShowingHandler(e) {
                const overlayContent = e.component.content().parentElement;
                overlayContent.setAttribute("role", "dialog");
            },
        },
    };
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';
    import Popover from 'devextreme-react/popover';

    const onShowingHandler = (e) => {
        const overlayContent = e.component.content().parentElement;
        overlayContent.setAttribute("role", "dialog");
    };

    const App = () => {
        return (
            <Popover ...
                onShowing={onShowingHandler}
            />
        );
    };

    export default App;

---

> Accessibility readers do not notify users when the Popover is opened since it is not a focusable element. To receive reader notifications for an open dialog, programmatically focus on an element within Popover after its activation:

Reviewer: ```suggestion
Accessibility readers do not notify users when the Popover is opened since it is not a focusable element. To receive reader notifications for an open dialog, programmatically focus an element in Popover after its activation:
```
